EasyInventory/src/main/java/com/easyinventory/enterprise/service/EasyInventoryServiceIMPL.java

```diff
@@ -13,9 +15,11 @@
 @Service
 public class EasyInventoryServiceIMPL implements EasyInventoryService{
 
-    ItemRepository itemRepository;
+    private ItemRepository itemRepository;
```

Comment:
Based upon java standards, I opted to add a private access modifier to this variables, as they are only accessed in this class.

https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html

Reply:
Good idea, nice citation.